addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/core/LxControlDimmer.java

```diff
@@ -137,7 +137,7 @@ private Double mapLoxoneToOH(Double loxoneValue) {
             Double max = getMax();
             Double min = getMin();
             if (max != null && min != null) {
-                return (loxoneValue - min) * ((max - min) / 100);
```

Comment:
This formula is wrong, it should read:
return 100 * (loxoneValue - min) / (max - min);